Sprint-3/1-key-implement/1-get-angle-type.js

```diff
@@ -8,7 +8,13 @@
 // Then, write the next test! :) Go through this process until all the cases are implemented
 
 function getAngleType(angle) {
-   if (angle === 90) return "Right angle";
+  if (angle === 90) return "Right angle";
+  if (angle < 90) return "Acute angle"
+  if (angle > 90 && angle < 180) return "Obtuse angle"
+  if (angle === 180) return "Straight angle"
+  if (angle > 180 && angle < 360) return "Reflex angle"
+  if(angle === 360)  return "Full angle"
+  else return "please insert a valid angle (0-360)"
    // read to the end, complete line 36, then pass your test here
 }
```

Comment:
it is   good to  handle your  logic you can also use else if to  combine your logic and in the last result always return relavant message that would fit context of your function